### PR TITLE
Quasar v1 upgrade

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

**Comment by gannan08** [comment]
@aljones15 We can squash those last two commits.

**Comment by aljones15** [comment]
@gannan08 commits were squashed earlier @dlongley good job like the validation.

File: BrQPassword.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <template v-slot:append>
       <q-icon
         :name="visible ? hideIcon : showIcon"
-        class="cursor-pointer"
+        class="cursor-pointer toggle-icon"
```
Comment (gannan08) [inline BrQPassword.vue:22]: Do we need a custom css class? Can this be achieved by leveraging quasar? Would `q-ml-xs` suffice?

File: BrQPassword.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      visible: false
+    };
+  }
+};
+</script>
+
+<style scoped>
+
+.toggle-icon {
+    margin-left: 10px;
```
Comment (davidlehn) [inline BrQPassword.vue]:
```suggestion
  margin-left: 10px;
```

File: BrQPassword.vue (hunk 1 above)
Comment (JoshDunnDev) [reply BrQPassword.vue:22]: I tried that initially but I feel like 10px looks better than 8px. We can use q-ml-xs if you would prefer.

**Review by gannan08 — APPROVED** [review]

**Pull request merged**